src/calculateCowsAndBulls.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+function calculateCowsAndBulls(generatedNumber, numberFromUser) {
```

Comment:
this function not only calculates the result but also validates the inputted data. Would be better to split this logic